Game_Manual/Levels_and_Roles.md

```diff
@@ -0,0 +1,156 @@
+| 1 | Δ/week>=17.5  | - | >=65 | >=65 | - | -8<=bias<=8 | - | joined a cohort  |
+| 2 | >=150 | >=1000 | >=80 | >=80 | - | - | >=20 | -                          |
+| 3 | >=500 | >=1040 | >=85 | >=85 | - | - | >=60 | -                          |
+| 4 | >=750 | >=1080 | >=90 | >=90 | >=90 | - | >=90 | ? coach stat (TBD)      |
+| 5 | >=1000 | >=1100 | >=90 | >=90 | >=95 | - | >=120 | ? coach stat (TBD)    |
+
+#### Moving Up and Back through Levels
+
+Moving up a level is simple: as soon as a player has met **all** of the requirements, they automatically advance.
+
+Moving back a level is a longer process. In order for a player to move back a level, their stats have to drop below the requirement and _stay below_ for two consecutive active cycles. When a player in level N drops their stats below the level N requirement, they are "in the red" and they have 2 cycles to move their stats back into compliance with the level N requirements.
+
+For example, say Dora is a player at level 2, with an Elo of 1008, cultural health at 89% and team play at 92%. Then, at the end of cycle 4, Dora's cultural health dips to 76%. She still in level 2, but she is now "in the red".
+
+If Dora doesn't bring her cultural health back to 80% or above by the end of cycle 6, then she will move back to level 1. She will lose her level 2 abilities until her stats are once again good enough to advance to level 2.
+
+### Why levels?
+
+The LOS has levels for many reasons. Mostly, levels are useful to create a structure through which players can advance, giving direction and acknowledgement of real achievement.
+
+Levels support two growth trajectories for learners:
+
+- An arc from scaffolded structure towards informed agency
+- An arc from personal development towards community leadership
+
+When they start, players have few options. This limitation is actually a benefit, because it allows new players to focus intensely on their own development: establishing good learning habits, solidifying foundational technical and collaborative skills, developing a fluency with the language and norms of the learning collective.
+
+Players advance by taking on more difficult and challenging work. They push themselves to learn beyond their comfort and capacity, integrate those learnings, and repeat. There will always be greater and more interesting relevant challenges for them to face.
+
+As players advance, they earn more freedom to choose how to spend their time, what skills to focus on, and what work to pursue. With this status comes the accountability (and privilege) to mentor and aid others, to distribute and compound their learnings throughout the collective.
+
+## Roles
+
+There are 3 roles in the LOS.
+
+By default, every learner has the "Player" role.
+
+The other two roles must be earned by advancing to higher levels.
+
+#### Player
+
+Purpose:
+- A learning collective that helps themselves and others to find dignified livelihood
```

Comment:
this purpose reads a little awkwardly, I think because "themselves" doesn't have anything to refer to. Maybe something like, `A learning collective that helps all members find a dignified livelihood` would flow better?